Makefile

```diff
@@ -6,11 +6,13 @@ formatSources = "src"
 # Reformat all code
 .PHONY: fmt
 fmt:
+	python -m isort .
 	python -m black $(formatSources)
 
 check: lint
 
 # Check code formatting
 .PHONY: lint
 lint:
-	python.exe -m black --check $(formatSources)
+	python -m isort --check .
+	python -m black --check $(formatSources)
```

Comment:
Removing the `.exe` extension is an unrelated change, to avoid confusion during review.  I'm guessing it was necessary at one point, but clearly that is not the case now.